library/std/src/collections/hash/map.rs

```diff
@@ -34,8 +34,8 @@ use crate::sys;
 /// attacks such as HashDoS.
 ///
 /// The hashing algorithm can be replaced on a per-`HashMap` basis using the
-/// [`default`], [`with_hasher`], and [`with_capacity_and_hasher`] methods. Many
-/// alternative algorithms are available on crates.io, such as the [`fnv`] crate.
+/// [`default`], [`with_hasher`], and [`with_capacity_and_hasher`] methods. 
```

Comment:
The CI is failing because of the tidy check. It detected a trailing space here:
```suggestion
/// [`default`], [`with_hasher`], and [`with_capacity_and_hasher`] methods.
```